testsuite/integration-arquillian/tests/base/testsuites/base-suite

```diff
@@ -0,0 +1,47 @@
+account,4
```

Comment:
This is just an initial stab at distributing the tests, we can tune this more later.

Also, we can consider naming the groups something more sensible than 1,2,3,4,5,6.